src/Symfony/Component/AssetMapper/ImportMap/Resolver/JsDelivrEsmResolver.php

```diff
@@ -26,7 +26,7 @@ final class JsDelivrEsmResolver implements PackageResolverInterface
     public const URL_PATTERN_DIST = self::URL_PATTERN_DIST_CSS.'/+esm';
     public const URL_PATTERN_ENTRYPOINT = 'https://data.jsdelivr.com/v1/packages/npm/%s@%s/entrypoints';
 
-    public const IMPORT_REGEX = '{from"/npm/((?:@[^/]+/)?[^@]+?)(?:@([^/]+))?((?:/[^/]+)*?)/\+esm"}';
+    public const IMPORT_REGEX = '#(?:import|export)\s*(?:\{[^}]*\}|\w+)?\s*\**[from]*("/npm/((?:@[^/]+/)?[^@]+?)(?:@([^/]+))?((?:/[^/]+)*?)/\+esm")#';
```

Comment:
This regex looks also from as it would match without the `from` keyword when specifying the binding for the import as well, which is wrong.

I think the regex should look like that:

```
(?:import\s*(?:(?:\{[^}]*\}|\w+|\*\s*as\s+\w+)\s*\bfrom\s*)?|export\s*(?:\{[^}]*\}|\*)\s*from\s*)("/npm/((?:@[^/]+/)?[^@]+?)(?:@([^/]+))?((?:/[^/]+)*?)/\+esm")
```

Reply:
I don't think we have the burden of catching all the edge cases, as we are guaranteed some regularity from jsdelivr. My regex incorrectly matches the invalid `import { something } "/npm/@some/package/+esm"` but your regex is smart enough to NOT match this. But jsdelivr won't serve invalid things like this to us.

Still, if we happen to make the regex smarter, I don't see any downside. Thanks for the suggestion - i've just updated the regex (all tests still green)